src/grpcd/runtime.rs

```diff
@@ -324,17 +336,17 @@ impl From<StateReport> for farcaster::State {
     }
 }
 
-impl From<farcaster::HealthCheckSelector> for HealthCheckSelector {
-    fn from(s: farcaster::HealthCheckSelector) -> Self {
+impl From<farcaster::NetworkSelector> for HealthCheckSelector {
```

Comment:
Do we still need an `HealthCheckSelector` in `bus` module or should we also match the new gRPC structure?

Reply:
No, we still need it. I think in case it's also useful in the cli. The call does take very long if you apply it to all network options.